Documentation/subcommands/export.md

```diff
@@ -0,0 +1,25 @@
+# rkt export
+
+Export an App Container Image (.aci) from an exited pod
+
+```
+$ rkt export UUID .aci
+```
+
+## Options
+
+| Flag | Default | Options | Description |
+| --- | --- | --- | --- |
+| `--overwrite` |  `false` | `true` or `false` | Overwrite the output aci if it exists  |
+
+## Global options
```

Comment:
This section has been reworked on master: see other commands.

Also, the PR description should be updated (it says the docs are still to come).